tracks/ruby/exercises/acronym/mentoring.md

```diff
@@ -11,41 +11,55 @@ module Acronym
   end
 end
 ```
+
 with variations for the regex (see below).
 
 ## Common suggestions
-* Most first submissions split the phrase into words with `String#split`:
+
+- Most first submissions split the phrase into words with `String#split`:
+
   ```ruby
   phrase.split(/[ -]/)
   ```
+
   followed by operations to find the first letters.
   Variation: first `gsub` or `tr` the non-space delimiters, then `split`.
 
-In all these cases: it's recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`). (At this point in the track, it's more about experiencing the strengths of Ruby's built-in methods than making them find them. There's ample opportunity for research later on in the track.)
+In all these cases: it is recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, give them also the RegEx (`/\b[a-zA-Z]/`).
```

Comment:
```suggestion
In all these cases: it is recommended to suggest the `String#scan` method and, because the exercise is not about Regular Expressions, also give them the RegEx (`/\b[a-zA-Z]/`).
```

Reply:
Sure, but I also saw some `it's` changed to `it is`, so 🤷

Reply:
I considered that as well, but this is potentially a bit more complex as well.

Because: "suggest the banana and, maybe, ice cream" compared to "suggest the banana and, maybe, also ice cream", the also is really redundant here, though we have more words between the comma and the continuation.  I have to consider if the second is being suggested alternatively or additionally.  The also can be placed better, or may even be removed.